internal/metrics/metrics.go

```diff
@@ -46,6 +47,7 @@ const (
 	hostTagName           = "nodename"
 	lifecycleTagName      = "lifecycle"
 	sparkSchedulerName    = "spark-scheduler"
+	nodeZoneLabel         = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
this should have come from a k8s constant ideally

Reply:
there are none

Reply:
The only one I saw was in kubelet code

Reply:
https://github.com/kubernetes/kubernetes/blob/a7966022d78bce63fa8212ea2519c99f49c2570e/staging/src/k8s.io/api/core/v1/well_known_labels.go#L21

Reply:
Ah, i see the issue. They only recently moved it to core/v1 https://github.com/kubernetes/kubernetes/pull/73616/files#diff-2ddc1f695cd4e7d5ef9a2d93148eeca7

:D